syntax = "proto3";
package get_arp;

service get_arp {
    rpc Get_Arp_Info (arpRequest) returns (arpReply) {}
}

message arpRequest {
    string username = 1;
    string password = 2;
}

message arpReply {
    string arp_info = 1;
}